packages/testing/src/api/scenario.ts

```diff
@@ -37,10 +37,7 @@ export interface DefineScenario {
     ModelName extends string | number | symbol = string | number | symbol,
     KeyName extends string | number | symbol = string | number | symbol
   >(
-    scenario: Record<
-      ModelName,
-      Record<KeyName, A.Compute<PrismaCreateType['data']>>
-    >
+    scenario: Record<ModelName, Record<KeyName, A.Compute<PrismaCreateType>>>
```

Comment:
This is the update I did for scenario type. Notice I remove type from field `data`. Anything I missed here? @dac09

Reply:
Oh yes, looks ok to me!